inspirehep/records/marshmallow/base.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+from copy import deepcopy
+
+from inspire_dojson.utils import get_recid_from_ref, strip_empty_values
+from marshmallow import post_dump
+from marshmallow.schema import Schema
+
+
+class InspireDummyIncludeAllFieldsSchema(Schema):
+    """Ugly way to easily dump whole records without specifying every field manually"""
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._post_dumps = []
+
+    @post_dump(pass_original=True)
+    def include_original_fields(self, object, original_data):
+        for key, value in original_data.items():
+            if key not in object and key not in self.exclude:
+                object[key] = original_data[key]
+        for dump_func in self._post_dumps:
+            object = dump_func(object, original_data)
+        return strip_empty_values(object)
+
+
+class PopulateRecidMixin(object):
```

Comment:
I think both (`IncludeAll` too) should be `schema` or `mixin`if possible for consistency